src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -164,14 +165,473 @@ Sono state individuate due metriche principali:
+
+#show figure: set block(breakable: true)
+#figure(
+table(
+      columns: (2fr,7fr,1fr),
+      align: (center,left,center),
+      fill:(_,row) =>if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Codice test*],[*Descrizione*],[*Stato*],
+      ..test_di_sistema.map(item => (item.at(0),item.at(1),item.at(2))).flatten().map(item => [#item])
+),caption: "Test di sistema.")
+
+=== Tracciamento dei test di sistema
+#show figure: set block(breakable: true)
+#figure(
+table(
+      columns: (80pt,80pt),
+      align: (center,center),
+      fill:(_,row) =>if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Codice test*],[*Codice caso d'uso*],
+      ..test_di_sistema.map(item => (item.at(0),item.at(3))).flatten().map(item => [#item])
+),caption: "Tracciamento dei test di sistema.")
 
 
+== Liste di controllo
 
+Le liste di controllo sono uno strumento che i verificatori possono utilizzare al fine di individuare errori ricorrenti nella documentazione. Tali liste prevedono anche una descrizione del problema al fine di poter fornire una spiegazione della richiesta di cambiamenti durante la fase di revisione.
+Le liste di controllo possono essere aggiornate durante tutto il corso del progetto dai Verificatori, man mano che vengono notati errori ricorrenti.
 
+#pagebreak()
+=== Struttura dei documenti
+
+#figure(
+table(
+      columns: (1fr, 2fr),
+      inset: 10pt,
+      align: (center, left),
+      [*Aspetto*],[*Spiegazione*],
+      [Vuoti documentativi],[Tutte le sezioni che non si intende completare, e che quindi devono rimanere vuote, devono essere cancellate.],
+      [Didascalia assente],[Tutte le tabelle e le immagini devono avere una didascalia descrittiva.],
+      [Ripetizione di elementi ricorrenti],[Ogni elemento che viene usato più di una volta, soprattutto se si sa a priori che verrà usato più volte, deve essere creato da una funzione #glossary("Typst").],
+      [Ordine non alfabetico],[I nomi dei componenti devono essere riportati in ordine alfabetico, in qualsiasi documento dove compaiono come elenco non numerato.],
+      [Aggiornamento fantasma],[Ad ogni insieme di modifiche ai documenti che devono essere aggiunte assieme, deve corrispondere una riga nella tabella del changelog, con un univoco numero di versione.],
+      [Titolo principali in mezzo alla pagina],[Tutti i titoli principali devono iniziare la pagina nella quale vengono inseriti.],
+
+),
+caption: [Lista di controllo per la struttura dei documenti]
+)
 
+#pagebreak()
+=== Errori ortografici, di lingua italiana e di forma
+
+#figure(
+table(
+      columns: (1fr, 2fr),
+      inset: 10pt,
+      align: (center, left),
+      [*Aspetto*],[*Spiegazione*],
+      [Errori di battitura],[Gli errori di battitura, di distrazione o di ignoranza devono essere rimossi.],
```

Comment:
forse errori di ignoranze è un po troppo, 

potrebbe essere 

Errori Ortografici: gli errori di battitura, di distrazione, e gli errori di ortografia in generale devono essere rimossi.

Reply:
O anzi, dato che è per il Verifcatore ci troglierei il ruolo attivo di rimozione(e lo farei un po ovunque): 

Errori Ortografici: Controllare gli errori di battitura, di distrazione, e l'ortografia in generale.

Reply:
Sistemato con:
      [Errori di sintassi],[Gli errori di sintassi (battitura o distrazione) devono essere rimossi.],